src/pipelines/rust.rs

```diff
@@ -27,6 +27,8 @@ pub struct RustApp {
     id: Option<usize>,
     /// Runtime config.
     cfg: Arc<RtcBuild>,
+    /// Is this module main or a worker.
+    type_: RustAppType,
```

Comment:
Nit: I pretty strongly prefer the `r#type` pattern, instead of the `_` suffix. I know that is purely preferential, but that is what the language feature is there for `:)`.

Reply:
I would argue `app_type` is the best field name here...